firefox-ios/Client/pings.yaml

```diff
@@ -91,3 +91,26 @@ usage-reporting:
     include_info_sections: false
     ping_schedule:
       - baseline
+
+onboarding-opt-out:
+  description: |
+    This ping is submitted when a user opts out of 
+    usage-reporting during the onboarding process. 
+    It is sent immediately upon opting out of Telemetry/Usage Tracking (T/UT) 
+    at ToS consent. This ping will only ever be sent during the 
+    initial onboarding flow. 
+    If the global telemetry preference is disabled, this ping 
+    includes specific settings to ensure it can still be submitted.
+  include_client_id: true
```

Comment:
This should be `false`, we don't have an ID generated when this ping will be sent so it should not include it.